### DISCUSSION TITLE
IPED x Nudetective integration

### DISCUSSION THREAD

**Discussion by benevenuti15** [sepinf-inc/IPED#936] [Q&A]
I saw that the IPED has integration with the LED tool (Digital Evidence Locator). I would like to know if there is also some form of integration between the IPED and Nudetective tools?

**Comment by lfcnassif** [comment]
It doesn't. What NuDetective feature we don't have you would like to see integrated? Maybe it could be implemented internally...

Furthermore, since NuDetective is closed source, integration maybe could lead to licensing issues...

**Reply by benevenuti15** [reply]
Are LED and Nudetective the same tool? the module for IPED pornographic content detection uses the LED in an integrated way?

**Reply by wladimirleite** [reply]
> Are LED and Nudetective the same tool? 

No, they have similar features, but are different tools.

> The module for IPED pornographic content detection uses the LED in an integrated way?

Some features present in LED were added to IPED (e.g. DIE-nudity detection, video frames extraction, parser for e-Mule Known.met) and IPED can used LED's hash database. So part of LED's code (and/or ideas) were integrated into IPED, but there is no direct dependence (like a common library).

**Reply by benevenuti15** [reply]
It's great, thanks!
